base/services/w32time/timedate.h

```diff
@@ -0,0 +1,124 @@
+#ifndef _TIMEDATE_H
+#define _TIMEDATE_H
+
+#include <stdarg.h>
+
+#define _INC_WINDOWS
+#define COM_NO_WINDOWS_H
+#define WIN32_NO_STATUS
+
+#include <windef.h>
+#include <winbase.h>
+#include <winnls.h>
+#include <winreg.h>
+#include <wingdi.h>
+#include <winuser.h>
+#include <wchar.h>
+#include <commctrl.h>
+#include <cpl.h>
+
+//#include "resource.h"
+
+#define MAX_KEY_LENGTH 255
+#define MAX_VALUE_NAME 16383
+#define SERVERLISTSIZE 6
+#define BUFSIZE 1024
+#define NTPPORT 123
+#define ID_TIMER 1
+
+typedef struct
+{
+  UINT idIcon;
+  UINT idName;
+  UINT idDescription;
+  APPLET_PROC AppletProc;
+} APPLET, *PAPPLET;
+
+extern HINSTANCE hApplet;
+
+
+/* dateandtime.c */
+INT_PTR CALLBACK DateTimePageProc(HWND hwndDlg, UINT uMsg, WPARAM wParam, LPARAM lParam);
+BOOL SystemSetTime(LPSYSTEMTIME lpSystemTime, BOOL SystemTime);
+
+
+/* timezone.c */
+INT_PTR CALLBACK TimeZonePageProc(HWND hwndDlg, UINT uMsg, WPARAM wParam, LPARAM lParam);
+
+
+/* internettime.c */
+INT_PTR CALLBACK InetTimePageProc(HWND hwndDlg, UINT uMsg, WPARAM wParam, LPARAM lParam);
+
+
+/* timedate.c */
+#if DBG
+VOID DisplayWin32ErrorDbg(DWORD dwErrorCode, const char *file, int line);
+#define DisplayWin32Error(e) DisplayWin32ErrorDbg(e, __FILE__, __LINE__);
+#else
+VOID DisplayWin32Error(DWORD dwErrorCode);
+#endif
```

Comment:
Unindented chunk of code.

Reply:
#if/#else isn't usually indented

Reply:
This whole file was shamelessly copied from "dll/cpl/timedate/timedate.h" in its entirety and then I commented out a reference to "resource.h" because it was not needed here. Is there a better way to handle this rather than just doing a copy?

Reply:
@ThFabba Is there a reason why #if/#else preprocessors directives aren't indented? I've seen some people actually do indent those for the sake of clean readability albeit they're just preprocessors and not actual program statements. Would be glad if you can clear this doubt for me! :)

Reply:
I don't think there's a particular reason, it's just not extremely common to indent them. Either way of doing it is fine.